data/records/lhcb-ntuples-hackathon.json

```diff
@@ -0,0 +1,1022 @@
+[
+  {
+    "abstract": {
+      "description": "<p>Data from proton-proton (pp) collisions collected by the LHCb experiment filtered to produce Ntuples for {insert reason for request}.</p><p>FIXME Here should come a long explanation of inputs, ntuples, etc.</p><p>FIXME If we can attach the <code>info.yaml</code> etc, this is the place wher they would be added."
```

Comment:
RE: Here should come a long explanation of inputs, ntuples, etc. 

This could be pretty tricky depending on how detailed we are. If we endeavor to provide a similar level of detail on the inputs as https://opendata.cern.ch/record/28071, there is a bit of groundwork to do (e.g. steps for how DST files were produced). At the very least we can point to the proper DST files used to create the Ntuples, as well as software versions used in the Ntuple Making step.

Reply:
The explanation can stay more brief, anything you would like to highlight and/or make available for search.